opsdroid/testing/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Testing helpers for opsdroid.
+
+opsdroid provides a set of pytest fixtures and other helpers for writing tests
+for both opsdroid core and skills.
+"""
+from .helpers import *  # noqa
```

Comment:
Why have a separate `helpers` file?

Reply:
I was thinking it would be good to have the externalapi helper and other larger blocks of code in a separate file to more core fixtures etc.?

Reply:
Maybe this should be in a `fixtures` file :thinking:

Reply:
👍 on a fixtures file

Reply:
done.